### PR TITLE
python 3 support changes

### PR DESCRIPTION
**Pull request by seanwiseman**

### CODE REVIEW

File: project_tests.sh [hunk 1]
```diff
@@ -1,9 +1,7 @@
...
 
-virtualenv venv
-source venv/bin/activate
-pip install -r requirements.txt
+tox
+. .tox/py35/bin/activate
 pip install coveralls
-coverage run -m unittest discover tests
-COVERALLS_REPO_TOKEN=$(cat /etc/coveralls/tokens/elife-crossref-xml-generation) coveralls
+COVERALLS_REPO_TOKEN=$(cat /etc/coveralls/tokens/elife-article) coveralls
```
Comment (gnott) [inline project_tests.sh]: This should maybe still use the ``elife-crossref-xml-generation`` coveralls token instead of the ``elife-article`` one?

**Comment by gnott** [comment]
Not a requirement here, but the tests seem to run faster when I added this to requirements.txt

```
coverage==4.4.2
```

It was using coverage 3.7.1 here but maybe because I had some older virtualenvs. It seems to work fine with coverage 4.4.2 so we could specify the version to use in requirements.txt, here and in all other repos?

**Comment by seanwiseman** [comment]
Thanks @gnott, noted regarding `coverage` version. We don't define this in the project requirements.txt, I will review where this is defined and see if we can bump the version.

**Comment by gnott** [comment]
That'd be super if you would track down the best place for coverage to be specified @seanwiseman. I ran into it not being available at all in https://github.com/elifesciences/package-poa  when I attempted to add Python 3 support. It seemed like when it tries to run tox, coverage isn't available when it tries to run tests. It might be another simple example to figure out where to specify it.

**Comment by seanwiseman** [comment]
Upon looking at a few projects, `coverage` 'is' sometimes defined in the `requirements.txt` and sometimes not, no real pattern. I think this is another task in its self and falls into unifying the methods in which we structure/test/package python projects. I will head this up.

**Pull request merged**